Example Apps/Example Swift/MenuManager.swift

```diff
@@ -115,32 +132,21 @@ private extension MenuManager {
     ///
     /// - Parameter manager: The SDL Manager
     /// - Returns: A SDLMenuCell object
-    class func menuCellChangeTemplate(with manager: SDLManager) -> SDLMenuCell {
+    func menuCellChangeTemplate(with manager: SDLManager) -> SDLMenuCell {
     
         /// Lets give an example of 2 templates
         var submenuItems = [SDLMenuCell]()
-        let errorMessage = "Changing the template failed"
         
         /// Non-Media
         let submenuTitleNonMedia = "Non - Media (Default)"
-        submenuItems.append(SDLMenuCell(title: submenuTitleNonMedia, icon: nil, voiceCommands: nil, handler: { (triggerSource) in
-            manager.screenManager.changeLayout(SDLTemplateConfiguration(predefinedLayout: .nonMedia)) { err in
-                if err != nil {
-                    AlertManager.sendAlert(textField1: errorMessage, sdlManager: manager)
-                    return
-                }
-            }
+        submenuItems.append(SDLMenuCell(title: submenuTitleNonMedia, icon: nil, voiceCommands: nil, handler: { [weak self] (triggerSource) in
+            self?.currentTemplate = .nonMedia
```

Comment:
I don't think we need to save the current template on both the menu manager and the proxy manager. If you want to keep the current template state on the proxy manager, then I would remove the `currentTemplate` variable and just call the `refreshTemplateHandler` handler here.